doc/build-windows-mingw.md

```diff
@@ -0,0 +1,250 @@
+From a MinGW shell unpack the source archive, configure and make:
+
+
+cd /c/deps/
+tar xvfz db-4.8.30.NC.tar.gz
+cd db-4.8.30.NC/build_unix
+../dist/configure --enable-mingw --enable-cxx --disable-shared --disable-replication
+make
+
+
+
+2.3 Boost: http://sourceforge.net/projects/boost/files/boost/1.61.0/
+Download either the zip or the 7z archive, unpack boost inside your C:\deps folder, then bootstrap and compile 
+from a Windows command prompt:
+
+
+cd C:\deps\boost_1_61_0\
+bootstrap.bat mingw
+b2 --build-type=complete --with-chrono --with-filesystem --with-program_options --with-system --with-thread toolset=gcc variant=release link=static threading=multi runtime-link=static stage
+
+This will compile the required boost libraries and put them into the stage folder (C:\deps\boost_1_61_0\stage).
+Note: make sure you don't use tarballs, as unix EOL markers can break batch files.
+
+
+2.3.1  Libevent 2.0.22: https://sourceforge.net/projects/levent/files/release-2.0.22-stable/libevent-2.0.22-stable.tar.gz/download
+Download and unpack, then run from the mysys shell:
+
+tar -xvfz libevent-2.022-stable.tar.gz 
+mv ./libevent-2.022-stable ./libevent-2.022
+
+cd libevent-2.022
+./configure --disable-shared
+
+make
+
+
+
+2.4 Miniupnpc: http://miniupnp.free.fr/files/download.php?file=miniupnpc-1.9.20151008.tar.gz
+Unpack Miniupnpc to C:\deps, rename containing folder from "miniupnpc-1.9.20151008" to "miniupnpc" then 
+from a Windows command prompt:
+
+
+cd C:\deps\miniupnpc
+mingw32-make -f Makefile.mingw init upnpc-static
+
+
+
+2.5 protoc and libprotobuf:
+Download https://github.com/google/protobuf/releases/download/v2.6.1/protobuf-2.6.1.tar.gz
+Then from msys shell:
+
+
+tar xvfz protobuf-2.6.1.tar.gz
+cd /c/deps/protobuf-2.6.1
+configure --disable-shared
+make
+
+
+
+2.6 qrencode:
+Download and unpack http://download.sourceforge.net/libpng/libpng-1.6.16.tar.gz inside your deps folder
+then configure and make:
+
+
+cd /c/deps/libpng-1.6.16
+configure --disable-shared
+make
+cp .libs/libpng16.a .libs/libpng.a
+
+
+
+Download and unpack http://fukuchi.org/works/qrencode/qrencode-3.4.4.tar.gz inside your deps folder 
+then configure and make:
+
+
+cd /c/deps/qrencode-3.4.4
+
+LIBS="../libpng-1.6.16/.libs/libpng.a ../../mingw32/i686-w64-mingw32/lib/libz.a" \
+png_CFLAGS="-I../libpng-1.6.16" \
+png_LIBS="-L../libpng-1.6.16/.libs" \
+configure --enable-static --disable-shared --without-tools
+
+make
+
+
+
+2.7 Qt 5 libraries:
+Qt must be configured with ssl and zlib support.
+Download and unpack Qt base and tools sources:
+http://download.qt-project.org/official_releases/qt/5.3/5.3.2/submodules/qtbase-opensource-src-5.3.2.7z
+http://download.qt-project.org/official_releases/qt/5.3/5.3.2/submodules/qttools-opensource-src-5.3.2.7z
+Then from a windows command prompt (note that the following assumes qtbase has been unpacked to C:\Qt\5.3.2 and 
+qttools have been unpacked to C:\Qt\qttools-opensource-src-5.3.2):
```

Comment:
Inconsistent root location for Qt here versus the bitcoin `configure` call below.  I believe the other path makes more sense as it keeps all dependencies together under the `C:\deps\` folder.